README.md

```diff
@@ -32,11 +34,9 @@ The API provides access to the following data collections in your account via RE
 * [Statuses](sections/user-statuses.md) (_deprecated_)
 * [Time & Expense Approvals](sections/approvals.md)
 * [Holidays](sections/holidays.md)
-
-## Pro & Enterprise Only
-
 * [Custom Fields](sections/custom-fields.md)
 * [Reports](sections/reports.md)
+* (**New!**) [Webhooks](sections/webhooks.md)
```

Comment:
Should this be its own top-level section? Right now its a bit lost in here...

Reply:
Done.